kyma-project/busola
# fix: navigate to namespace with limited access

**Description**

Changes proposed in this pull request:

- fix navigating to namespace with limited access
- fix in displaying `IncorrectPath` component for a second when refreshing extensibility views
- add test

How to test?
- Create a namespace only user with `cluster-admin` role
- Upload a new kubeconfig
- Go to e.g. Deployments view
- On main, Incorrect Path is shown but shouldn't

**Related issue(s)**
